database/seeds/fill_class_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class fill_class_table extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('class')->insert([
+            'name' => 'Fullstack Web Developer 2019'
+        ]);
+
+        DB::table('class')->insert([
+            'name' => 'Fullstack Web Developer 2020'
+        ]);
+
+        DB::table('class')->insert([
+            'name' => 'Interaktionsdesigner 2019'
+        ]);
+
+        DB::table('class')->insert([
+            'name' => 'Interaktionsdesigner 2020',
+        ]);
```

Comment:
Might be nice to shorten the names in the DB. Like this: FWD19 & IK19.
What do you think?

Reply:
Yep, then we can just check for the name and write the full name in blade if we need to